cmd/algokey/keyreg.go

```diff
@@ -173,7 +173,7 @@ func run(params keyregCmdParams) error {
 	}
 
 	if util.FileExists(params.txFile) || params.txFile == stdoutFilenameValue {
-		return fmt.Errorf("outputFile '%s' already exists", params.partkeyFile)
+		return fmt.Errorf("outputFile '%s' already exists", params.txFile)
```

Comment:
This error message makes sense only for the first case of the conditional. In the second case, you might want to have a separate error message.

Reply:
Looks like that expression is wrong. It should be verifying that it is not the stdout filename.